### DISCUSSION TITLE
Chenges

### DISCUSSION THREAD

**Discussion by Hanseman66** [klaasnicolaas/home-assistant-glow#219] [Q&A] [answered]
Hi All 
In Enegy i can se Energy Distribution. 
Now it shows week . 
Can that be change to what im using now ? 

I have used Glow  for both solar panels and electrical input 

Br 
Hans

**Comment by klaasnicolaas** [comment, marked as answer]
I'm not quite following exactly what you mean by "change to what im using now". Can you perhaps clarify?

**Reply by Hanseman66** [reply]
Hi 
I want to se the consumtion right now insted of the week.

**Reply by klaasnicolaas** [reply]
Current consumption is not covered by the energy dashboard, there is a difference between power (w) or energy (kWh), [read also this](https://www.home-assistant.io/docs/energy/faq/#energy-vs-power). If you really want to get started with the real-time data, you could make something yourself in a dashboard.

**Comment by Hanseman66** [comment]
Hi 
Thanks for yor answer. 
I'm new to this homeautomation, so i guess i have some catchup to do before i can make somthing by my self. ... 

Br
Hans
